daemon/src/taker_cfd.rs

```diff
@@ -483,12 +479,6 @@ where
             .await?
             .with_context(|| format!("Announcement {} not found", cfd.order.oracle_event_id))?;
 
-        self.oracle_actor
```

Comment:
This was removed because it is just a weird duplicate that should not have been here. we trigger monitoring for attestation upon contract-setup/rollover completion, not when an order is accepted...